languages/common-lisp/bin/generate-scaffolding/template/.meta/config.json

```diff
@@ -7,8 +7,8 @@
   ],
   "contributors": [],
   "forked_from": [],
-  "editor": {
-    "solution_files": ["SLUG.lisp"],
-    "test_files": ["SLUG-test.lisp"]
+  "files": {
+    "solution": ["SLUG.lisp"],
+    "test": ["SLUG-test.lisp"]
   }
```

Comment:
I know that this is part of common-lisp's concept exercise generator rather than an actual concept exercise - but the script you ran didn't seem to add `examplar` here... If too annoying to fix in this PR we can fix it in the track later.

Reply:
I've added it manually.